api/app/Models/User.php

```diff
@@ -1351,7 +1353,7 @@ public static function hydrateSnapshot(mixed $snapshot): User|array
         $user = self::hydrateFields($snapshot, $fields, $user);
 
         if (isset($snapshot['department'])) {
-            $user->department = $snapshot['department']['id'];
+            $user->computed_department = $snapshot['department']['id'];
         }
 
         return $user;
```

Comment:
Department appears now, but classification is still missing from the application doc download. I'm not sure where the issue lies, maybe in `hydrateSnapshot`?
Profile doc download and application CSV however appear to work fine

Application doc download, government information
![image](https://github.com/user-attachments/assets/6525d548-ba6a-414f-a7e9-45b37dd1e406)

Reply:
Honestly a little confused as to how this was working on main :sweat_smile: Anyway, should be fixed after [fix classification hydration](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12653/commits/aa19177e76f032a86c034db8ab3a90455d8f2368)